src/ceph_osd.cc

```diff
@@ -489,19 +489,26 @@ int main(int argc, const char **argv)
   Messenger *ms_hb_front_client = Messenger::create(g_ceph_context, public_msg_type,
 					     entity_name_t::OSD(whoami), "hb_front_client",
 					     getpid(), Messenger::HEARTBEAT);
-  Messenger *ms_hb_back_server = Messenger::create(g_ceph_context, cluster_msg_type,
-						   entity_name_t::OSD(whoami), "hb_back_server",
+  Messenger *ms_hb_front_client_legacy = Messenger::create(g_ceph_context,
+							   public_msg_type,
+							   entity_name_t::OSD(whoami),
+							   "hb_front_client_legacy",
+							   getpid(), Messenger::HEARTBEAT);
+  Messenger *ms_hb_back_server = Messenger::create(g_ceph_context, cluster_msgr_type,
```

Comment:
variable name typo (probably due to last rebase): `cluster_msgr_type` should be `cluster_msg_type`